field_rrule/demo/res_partner.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could we examine the `config` command line options to check if `--demo` was used?